### PR TITLE
Add CI and example data

### LINKED ISSUE #27: ENH: Add pathogen-ci to ci
### Context
Pathogen CI has become quite useful to test compatibility of new augur features, and to check that docker images and conda environments are compatible.

It would be nice to add example data so that this workflow can be easily tested within the remit of pathogen ci.

### PR DESCRIPTION
**Pull request by victorlin**
### Description of proposed changes

Generally inspired by existing practices in the monkeypox repo.

### Related issue(s)

- Closes #27

### Testing

- [x] Checks pass

### Post-merge

Add this repo to pathogen repo CI lists:

- [ ] [Augur](https://github.com/nextstrain/augur/blob/7766f46c90e88d46a94ca7a6cf9cd2bf4291e2a8/.github/workflows/ci.yaml#L91-L104)
- [ ] [conda-base](https://github.com/nextstrain/conda-base/blob/a8bfdd21763259127307b796b30e623a32b11b16/.github/workflows/ci.yaml#L128-L135)
- [ ] [docker-base](https://github.com/nextstrain/docker-base/blob/8bb8465ccc36cb3eecd075c9219cdb2bae7d7b11/.github/workflows/ci.yml#L213-L220)

### CODE REVIEW

File: workflow/snakemake_rules/chores.smk [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Copied from the monkeypox repo¹, with RSV-specific edits.
+# ¹ https://github.com/nextstrain/monkeypox/blob/5c461dc7e90cd70c1f16b193f82fd1666d4c95e2/workflow/snakemake_rules/chores.smk
+
+rule update_example_data_wildcards:
+    """This updates the files under example_data/ based on latest available data from data.nextstrain.org.
+
+    The subset of data is generated by an augur filter call which:
+    - sets the subsampling size to 50
```
Comment (victorlin) [inline workflow/snakemake_rules/chores.smk:8]: To reviewers: Does anyone know if there are filter options to better subsample the RSV data? The [monkeypox filter](https://github.com/nextstrain/monkeypox/blob/5c461dc7e90cd70c1f16b193f82fd1666d4c95e2/workflow/snakemake_rules/chores.smk#L26-L27) includes the root and groups by clade + lineage. I'm not sure if those apply here.

Comment (j23414) [reply workflow/snakemake_rules/chores.smk:8]: Unfortunately I'm not seeing 'clade', 'lineage' or similar in the metadata:

```
$ head -n1 data/*/metadata.tsv                             
==> data/a/metadata.tsv <==
accession	genbank_accession_rev	strain	date	region	country	division	location	host	date_submitted	sra_accession	abbr_authors	reverse	authors	institution	F_coverage	G_coverage	genome_coverage

==> data/b/metadata.tsv <==
accession	genbank_accession_rev	strain	date	region	country	division	location	host	date_submitted	sra_accession	abbr_authors	reverse	authors	institution	F_coverage	G_coverage	genome_coverage
```

which would be required for the monkeypox-like filter. Clades aren't currently part of [ingest rules](https://github.com/nextstrain/rsv/tree/master/ingest/workflow/snakemake_rules), and doesn't happen until [here](https://github.com/nextstrain/rsv/blob/master/workflow/snakemake_rules/clades.smk) in the build. Maybe a way forward is adding clade assignment in ingest but I would imagine this as a separate discussion and PR.

To get at your "better subsampling" question, maybe group by year and country? Similar to the rsv config file?

https://github.com/nextstrain/rsv/blob/ba171f4a43110382c38b6154be3febd50408d7bf/config/configfile.yaml#L16-L17

Others, feel free to chime in.

Comment (victorlin) [reply workflow/snakemake_rules/chores.smk:8]: Thanks, that's a good suggestion! Updated to use that config directly in 538442d0f7ce34a748f19a672e0888f52f4786d8.

Comment (j23414) [reply workflow/snakemake_rules/chores.smk:8]: Nice! LGTM :D

**Comment by victorlin** [comment]
Merging since this is already useful in #29.

**Pull request merged**